src/vs/workbench/parts/debug/electron-browser/debugInlineDecorators.ts

```diff
@@ -0,0 +1,167 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IExpression } from 'vs/workbench/parts/debug/common/debug';
+import { IDecorationOptions, IRange, IModel } from 'vs/editor/common/editorCommon';
+import { createStringDictionary, IStringDictionary } from 'vs/base/common/collections';
+import { StandardTokenType } from 'vs/editor/common/core/lineTokens';
+
+export const MAX_INLINE_VALUE_LENGTH = 50; // Max string length of each inline 'x = y' string. If exceeded ... is added
+export const MAX_INLINE_DECORATOR_LENGTH = 150; // Max string length of each inline decorator when debugging. If exceeded ... is added
+export const MAX_NUM_INLINE_VALUES = 100; // JS Global scope can have 700+ entries. We want to limit ourselves for perf reasons
+export const MAX_TOKENIZATION_LINE_LEN = 500; // If line is too long, then inline values for the line are skipped
+export const ELLIPSES = '…';
+// LanguageConfigurationRegistry.getWordDefinition() return regexes that allow spaces and punctuation characters for languages like python
+// Using that approach is not viable so we are using a simple regex to look for word tokens.
+export const WORD_REGEXP = /[\$\_A-Za-z][\$\_A-Za-z0-9]*/g;
```

Comment:
\b doesn't work since it doesn't consider `$` and `_`  as `word` character. Powershell, php would break seriously. Javascript allows $ and _ in variable names so they would have quirks as well.